app/src/main/java/com/mapbox/services/android/navigation/testapp/example/utils/RouteProgressUtils.kt

```diff
@@ -4,12 +4,11 @@ import android.content.Context
 import android.text.format.DateFormat
 import com.mapbox.services.android.navigation.v5.navigation.NONE_SPECIFIED
 import com.mapbox.services.android.navigation.v5.routeprogress.RouteProgress
+import com.mapbox.services.android.navigation.v5.utils.extensions.ifNonNull
 import com.mapbox.services.android.navigation.v5.utils.time.TimeFormatter.formatTime
 import java.util.Calendar
-
-fun RouteProgress.formatArrivalTime(context: Context): String {
-    val time = Calendar.getInstance()
-    val isTwentyFourHourFormat = DateFormat.is24HourFormat(context)
-    // TODO localization 12 /24 time specify from settings
-    return formatTime(time, durationRemaining(), NONE_SPECIFIED, isTwentyFourHourFormat)
-}
+fun RouteProgress.formatArrivalTime(context: Context): String =
```

Comment:
better to move it to `utils.extensions` folder

Reply:
I did not create the file. I just modified. The author of the file should move it if that is what's desired